orchestrator/careplanservice/handle_getservicerequest.go

```diff
@@ -29,17 +30,17 @@ func (s *Service) handleGetServiceRequest(ctx context.Context, id string, header
 	if err != nil {
 		return nil, err
 	}
-	// If bundle is empty, the user does not have access to the ServiceRequest
+	// If the user does not have access to the Task, check if they are the creator of the ServiceRequest
```

Comment:
Ah right, now I understand. Access is allowed if either:
- The principal has access to the Task (through CareTeam or being Task.request/owner), or
- The principal is the creator of the resource